dapps/marketplace/src/components/TelegramLoginButton.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, memo } from 'react'
+
+const TelegramLoginButton = ({ redirectURL, buttonText, className }) => {
+  let rootEl
+
+  useEffect(() => {
+    const script = document.createElement('script')
+    script.async = true
+    script.setAttribute('src', 'https://telegram.org/js/telegram-widget.js?7')
+    script.setAttribute(
+      'data-telegram-login',
+      process.env.TELEGRAM_BOT_USERNAME
```

Comment:
Can we return an error if this is not set?

Reply:
Can do that